### PR TITLE
Devex pdf cleanup 1

### PR DESCRIPTION
**Pull request by matthopson**

### CODE REVIEW

File: shared/src/business/utilities/generateHTMLTemplateForPDF/reactTemplateGenerator.js [hunk 1]
```diff
@@ -3,15 +3,19 @@ require('@babel/register')({
...
 
-const ChangeOfAddress = require('../pdfGenerator/documentTemplates/ChangeOfAddress.jsx')
-  .default;
-const DocketRecord = require('../pdfGenerator/documentTemplates/DocketRecord.jsx')
-  .default;
-const StandingPretrialOrder = require('../pdfGenerator/documentTemplates/StandingPretrialOrder.jsx')
-  .default;
+const {
+  ChangeOfAddress,
+} = require('../pdfGenerator/documentTemplates/ChangeOfAddress.jsx');
```
Comment (rachaelparris) [inline shared/src/business/utilities/generateHTMLTemplateForPDF/reactTemplateGenerator.js:8]: ❤️

File: shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx [hunk 2]
```diff
@@ -75,7 +76,49 @@ const RenderPractitioner = ({
...
+    <>
+      <strong>{record.description}</strong>
+      {additionalDescription}
+    </>
+  );
+};
+
+const ServedDate = ({ document }) => {
+  const documentDateServed =
+    document && document.isStatusServed ? document.servedAtFormatted : '';
```
Comment (rachaelparris) [inline shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx:98]: I'd love to see this sort of logic removed from the views later on.

**Review by rachaelparris — APPROVED** [review]
Beautiful.

File: shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx (hunk 2 above)
Comment (matthopson) [reply shared/src/business/utilities/pdfGenerator/documentTemplates/DocketRecord.jsx:98]: Yeah, I've been trying to decide how to go about that. Keeping the components as dumb as possible, I think we'd want to do something similar to the computeds, perhaps in the documentGenerator that's calling the component - but I didn't want that code to get to messy either.

**Pull request merged**